scripts/zones/Valkurm_Dunes/npcs/Nyata-Mobuta_WW.lua

```diff
@@ -15,6 +16,11 @@ local guardEvent  = 32759
 
 entity.onTrade = function(player, npc, trade)
     xi.conq.overseerOnTrade(player, npc, trade, guardNation, guardType)
+    if(player:getNation()==guardNation) then
```

Comment:
no need for parenthesis + spacing in conditional

player:getNation() == guardNation